frontend/src/components/annotation/RadioInput.vue

```diff
@@ -11,7 +11,8 @@
         :inline="config.orientation!=='vertical'"
         :name="config.name"
         >
-          {{ option.text }}
+          <span v-if="option.html" v-html="option.html"></span>
+          <span v-else>{{ option.text }}</span>
```

Comment:
This is the core of the changes - if there's an HTML label then bind that with `v-html`, otherwise use the text label as before.